backend/tests/conftest.py

```diff
@@ -121,6 +126,61 @@ def mock_bitquery_api_get_blocks_rewards(*args, **kwargs):
     return example_resp_json["data"]["ethereum"]["blocks"][0]["reward"]
 
 
+def mock_safe_api_message_details(*args, **kwargs):
```

Comment:
`conftest` file is growing enormously, we should split it into chunks. Could you create a task to do that?

Reply:
agree on it, i have already created one